pkg/policy/updaterequest.go

```diff
@@ -14,6 +14,13 @@ import (
 
 func (pc *PolicyController) updateUR(policyKey string, policy kyverno.PolicyInterface) {
 	logger := pc.log.WithName("updateUR").WithName(policyKey)
+
+	// TODO: add check for genExisting
```

Comment:
@prateekpandey14 - we also need to add the check for `genExisting` here.

Reply:
Sure

Reply:
GenExisting is per rule level which is checked below, not at spec level.

Do we need to check the `MutateExistingOnPolicyUpdate` here, with this check we are changing the older behaviour of `UpdateUR()`